eva-lib/src/main/java/uk/ac/ebi/eva/lib/config/EvaProperty.java

```diff
@@ -0,0 +1,130 @@
+package uk.ac.ebi.eva.lib.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@ConfigurationProperties(prefix = "eva")
+@Component
+public class EvaProperty {
```

Comment:
`EvaProperties` would be more aligned with the class name from the standard Java API.